tasks/vendoring/patches/vendor/pipdeptree-update-pip-import.patch

```diff
@@ -1,13 +1,18 @@
-diff --git a/pipenv/vendor/pipdeptree.py b/pipenv/vendor/pipdeptree.py
-index 250366ec..af4be292 100644
---- a/pipenv/vendor/pipdeptree.py
-+++ b/pipenv/vendor/pipdeptree.py
-@@ -20,6 +20,8 @@ try:
- except ImportError:
-     from collections import Mapping
-
-+pardir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+diff --git a/pipenv/vendor/pipdeptree/__main__.py b/pipenv/vendor/pipdeptree/__main__.py
+index 85cca3c..a002019 100644
+--- a/pipenv/vendor/pipdeptree/__main__.py
++++ b/pipenv/vendor/pipdeptree/__main__.py
+@@ -1,5 +1,13 @@
++import os
+ import sys
+ 
++pardir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
++# for finding pipdeptree itself
```

Comment:
I bet we could drop this patch and add `pipdeptree` to the vendoring tasks `LIBRARY_RENAMES` and re-run vendoring to get the imports corrected.

Reply:
I tried this, without much success. We call this tool with a subshell, so all the paths are weird. 

I'd rather not invest more time on this now.